<?xml version="1.0"?>
<launch>
  <arg name="config_file_path" default="$(find smb_mission_planner)/configs/examples/my_config.yaml"/>
  <arg name="waypoint_topic_name" default="/move_base_simple/goal"/>
  <arg name="base_pose_topic_name" default="/base_pose_measured"/>

  <node pkg="smb_mission_planner"
        type="mission_recorder_node.py"
        name="mission_recorder_node"
        output="screen"
        args="$(arg config_file_path) $(arg waypoint_topic_name) $(arg base_pose_topic_name)">
  </node>

</launch>
